src/components/composer/composer/composer.ts

```diff
@@ -169,7 +169,7 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
       let assistantStyle: CSSProperties = {};
       if (cellY > remainingHeight) {
         // render top
-        assistantStyle.top = `$-3px`;
+        assistantStyle.top = `-3px`;
```

Comment:
I invistigated a bit the `-3px` because why -3. I think the -3 is because : -2px to compensate the vertical margin of the `o-composer-assisstant`, and -1px to be displayed a px above the border of the composer.

If we replace the margin of the `o-composer-assistant` with padding, we don't need the -2.
The -1 for the border is unavoidable I think, but it's a bit strange since we don't do that when the assisstant is below the composer, the composer border will be hidden by the assisstant. 

We should at the very least add a comment on the why this value.